usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -102,6 +105,23 @@
         "column_names": list(TRANSACTION_FABS_COLUMNS),
         "tsvectors": None,
     },
+    "published_fabs": {
+        "model": SourceAssistanceTransaction,
+        "is_from_broker": False,
+        "source_table": "published_fabs",
```

Comment:
The USAS table is `source_assistance_transaction`

Reply:
addressed